docs/docs/Develop/logging.mdx

```diff
@@ -9,46 +9,86 @@ import TabItem from '@theme/TabItem';
 
 This page provides information about Langflow logs, including logs for individual flows and the Langflow application itself.
 
-Langflow uses the `loguru` library for logging.
+Langflow uses the [structlog](https://www.structlog.org) library for logging.
 
 The default logfile is named `langflow.log`.
 Langflow also produces logfiles for flows.
 
+Log files are stored in JSON format with structured metadata.
+
 ## Log storage
 
-The default logfile storage location depends on your operating system:
+Langflow logs are stored in the config directory specified in the `LANGFLOW_CONFIG_DIR` environment variable.
+The default config directory location depends on your operating system and installation method:
 
-- **macOS Desktop**:`/Users/<username>/.langflow/cache`
+- **macOS Desktop**:`/Users/<username>/Library/Logs/com.Langflow`
+- **macOS OSS**:`/Users/<username>/Library/Caches/langflow`
 - **Windows Desktop**:`C:\Users\<username>\AppData\Roaming\com.Langflow\cache`
```

Comment:
2 comments:

1. Missing path for windows/*nix OSS?
2. Could use a a partial for these lines (21-26), then import it to both logging.mdx and concepts-flows.mdx?